src/ets/scala/io/iohk/ethereum/ets/blockchain/ScenarioSetup.scala

```diff
@@ -1,35 +1,43 @@
 package io.iohk.ethereum.ets.blockchain
 
 import io.iohk.ethereum.blockchain.sync.EphemBlockchainTestSetup
-import io.iohk.ethereum.consensus.{ConsensusBuilder, ConsensusConfigBuilder}
+import io.iohk.ethereum.consensus.ethash.validators.EthashBlockHeaderValidator
 import io.iohk.ethereum.domain.Block.BlockDec
 import io.iohk.ethereum.domain.{Account, Address, Block, UInt256}
 import io.iohk.ethereum.ets.common.AccountState
 import io.iohk.ethereum.ledger.Ledger.VMImpl
 import io.iohk.ethereum.ledger._
 import io.iohk.ethereum.network.p2p.messages.PV62.BlockBody
-import io.iohk.ethereum.nodebuilder.{ActorSystemBuilder, BlockchainConfigBuilder, ShutdownHookBuilder, SyncConfigBuilder, ValidatorsBuilder}
+import io.iohk.ethereum.nodebuilder._
 import io.iohk.ethereum.utils.BigIntExtensionMethods._
-import io.iohk.ethereum.utils.{BlockchainConfig, Logger}
+import io.iohk.ethereum.utils.BlockchainConfig
+import io.iohk.ethereum.validators._
 import org.spongycastle.util.encoders.Hex
 
 import scala.util.{Failure, Success, Try}
 
+object ScenarioSetup {
+  // FIXME: this is a temporary (hopefully) fix for BlockchainSuite. It avoids using `ValidatorsBuilder` which causes
+  // the tests to run into OOM errors (for reason currently unknown)
```

Comment:
This is worth getting to the bottom of. It's been floating around for a while. It's trying to tell us something (!!) Potentially something to do with lazy init.... :thinking:

Reply:
> It's been floating around for a while.

I am not aware of any previous incidents when `ValidatorsBuilder` was causing an OOM. This was introduced in #407.
Or do you mean OOM errors in general?

Reply:
I mean OOM in general. I *think* I've noticed an uptick in occurrences. Although I've no hard facts to back that up. I could be wrong?

Reply:
Well, this will always be a memory hungry beast. And the reasons for going OOM vary case by case, I think.